### PR TITLE
logging config, refactoring, more handler calls

### PR DESCRIPTION
**Pull request by e0d**

### CODE REVIEW

File: playbooks/roles/xqwatcher/tasks/deploy_watcher.yml [hunk 1]
```diff
@@ -7,26 +7,44 @@
...
+  template: >
+    src=edx/app/xqwatcher/xqwatcher.json.j2
+    dest={{ xqwatcher_conf_dir }}/xqwatcher.json
+    mode=0644 owner={{ xqwatcher_user }} group={{ xqwatcher_user }}
+  notify: restart xqwatcher
 
+- name: write out watcher logging config file
+  template: >
+    src=edx/app/xqwatcher/logging.json.j2
+    dest={{ xqwatcher_conf_dir }}/logging.json
```
Comment (feanil) [inline playbooks/roles/xqwatcher/tasks/deploy_watcher.yml:30]: I don't see a template for logging.json, was that added previously?

**Comment by hkim823** [comment]
@e0d wanted to check in on the status of this PR.  still needed?

File: playbooks/roles/xqwatcher/templates/edx/app/xqwatcher/logging.json.j2 [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            "requests": {
+                "level": "{{ xqwatcher_requests_log_level }}",
+                "handlers": ["console"]
+            }
+    },
+    "version": 1,
+    "handlers": {
+        "console": {
+            "class": "logging.StreamHandler",
+            "level": "DEBUG",
```
Comment (feanil) [inline playbooks/roles/xqwatcher/templates/edx/app/xqwatcher/logging.json.j2:16]: Should this also be set to `{{ xqwatcher_log_level }}`?

**Comment by feanil** [comment]
@e0d this needs a rebase and I have one comment on the logging.json but otherwise looks good.

File: playbooks/roles/xqwatcher/defaults/main.yml [hunk 3]
```diff
@@ -82,7 +82,8 @@ xqwatcher_git_identity: "{{ xqwatcher_app_dir }}/git-identity"
 
 xqwatcher_requirements_file: "{{ xqwatcher_code_dir }}/requirements.txt"
 xqwatcher_log_dir: "{{ COMMON_LOG_DIR }}/{{ xqwatcher_service_name }}"
-
+xqwatcher_log_level: "ERROR"
```
Comment (carsongee) [inline playbooks/roles/xqwatcher/defaults/main.yml:85]: Can we all caps these vars for modification?

**Pull request closed without merging**